diff -urNad conduit-0.3.17~/conduit/modules/RhythmboxModule/RhythmboxModule.py conduit-0.3.17/conduit/modules/RhythmboxModule/RhythmboxModule.py
--- conduit-0.3.17~/conduit/modules/RhythmboxModule/RhythmboxModule.py	2010-01-07 17:10:30.000000000 +0100
+++ conduit-0.3.17/conduit/modules/RhythmboxModule/RhythmboxModule.py	2010-03-28 14:37:12.034777215 +0200
@@ -50,8 +50,8 @@
     _icon_ = "rhythmbox"
     _configurable_ = True
 
-    PLAYLIST_PATH="~/.gnome2/rhythmbox/playlists.xml"
-    RHYTHMDB_PATH="~/.gnome2/rhythmbox/rhythmdb.xml"
+    PLAYLIST_PATH="~/.local/share/rhythmbox/playlists.xml"
+    RHYTHMDB_PATH="~/.local/share/rhythmbox/rhythmdb.xml"
 
     def __init__(self, *args):
         DataProvider.DataSource.__init__(self)
@@ -150,7 +150,7 @@
     '''Wrapper around the standard Audio datatype that implements
     the rating, playcount, and cover location tags.
     '''
-    COVER_ART_PATH="~/.gnome2/rhythmbox/covers/"
+    COVER_ART_PATH="~/.cache/rhythmbox/covers"
     def __init__(self, URI, **kwargs):
         Audio.Audio.__init__(self, URI, **kwargs)
         self._songdata = kwargs['songdata'] or {}
