diff -urNad conduit-fix~/conduit/modules/FileModule/FileModule.py conduit-fix/conduit/modules/FileModule/FileModule.py
--- conduit-fix~/conduit/modules/FileModule/FileModule.py	2009-02-24 22:56:27.000000000 +0100
+++ conduit-fix/conduit/modules/FileModule/FileModule.py	2009-02-24 23:03:47.000000000 +0100
@@ -116,7 +116,7 @@
         return self.folder
         
     def get_name(self):
-        return Vfs.uri_get_filename(self.folder)
+        return Vfs.uri_unescape(Vfs.uri_get_filename(self.folder))
 
     def _monitor_folder(self):
         self._monitor.add(self.folder, self._monitor.MONITOR_DIRECTORY)
